cves/CVE-2012-2779.yml

```diff
@@ -172,10 +184,14 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked. Thus, 'answer' should always have some
     explanation.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: | 
+    Looking through the commit, there is a mention that the vulnerability was 
+    found by two people. However, neither of those people were the original developer 
+    and did not contribute to the fix. So it is unknown how they found the vulnerability. 
+    My guess would be through some form of testing.
```

Comment:
Are there bugs referenced anywhere? Do the mailing lists have anything around these fix dates?